doc/deployment/deploy_custom.md

```diff
@@ -1,6 +1,118 @@
 # Custom Deployments
 
-This document  details the various options of the `pachctl deploy custom ...` command for an on-premises deployment. 
+If you're deploying to a cloud infrastructure, 
+such as [Amazon Web Services (AWS)](https://pachyderm.readthedocs.io/en/latest/deployment/amazon_web_services.html),
+[Google Cloud Platform (GCP)](https://pachyderm.readthedocs.io/en/latest/deployment/google_cloud_platform.html), or 
+[Microsoft Azure](https://pachyderm.readthedocs.io/en/latest/deployment/azure.html), 
+Pachyderm's `pachctl deploy` includes commands for those providers (`amazon`, `google`, and `microsoft`, respectively).
+You should use those commands when deploying to those cloud infrastructures.
+You can customize cloud provider deployments extensively through flags available for each provider.
+
+Pachyderm also includes `pachctl deploy custom` for creating customized deployments for cloud providers or on-premises use.
+It's usually invoked with the `--dry-run` flag, 
+with the command's standard output directed into a series of customization scripts or a file for editing.
+
+This document covers using `pachctl deploy custom ... --dry-run` to create a manifest for a custom, on-premises deployment.
+We won't directly address automating your deployment by means of customization scripts,
+but we do encourage you to treat your [infrastructure as code](./on-premises.html#infrastructure-as-code).
+
+## Anatomy of a Pachyderm deployment manifest
+
+When you run the `pachctl deploy ...` command with the `--dry-run` flag,
+you're generating a JSON-encoded Kubernetes manifest in one stream to standard output. 
+That manifest itself consists of a number of smaller manifests,
+corresponding to a particular aspect of a Pachyderm deployment.
+
+Pachyderm has three sets of application components usually deployed:
+- `pachd`, the main Pachyderm pod
+- `etcd`, the administrative datastore for `pachd`
+- `dash`, the web-based enterprise ui for Pachyderm
+We'll explore all of these within this document.
+
+In general, there are two categories of manifests in the file,
+roles-and-permissions-related and application-related.
+
+## Roles-and-permissions-related manifests
+
+### ServiceAccount
+
+Usually at the top of the file, this manifest has the `kind` key set to `ServiceAccount`. 
+[ServiceAccounts](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/) are a way Kubernetes can assign namespace-specific privileges to applications in a lightweight way.
+Pachyderm's service account is called, appropriately enough, `pachyderm`.
+
+### Role or ClusterRole
+
+The next manifest will usually be of `kind` `Role` or `ClusterRole`, 
+depending on whether you  used the `--local-roles` flag `pachctl deploy` command.
```

Comment:
sentence structure is weird here. Just remove "`pachctl deploy` command"

Reply:
Add: At a high level, what does "role" section of manifest define? 1 sentence is plenty.

Reply:
I believe this is missing a "to" before "`pachctl deploy` command", which would make it clearer, but I'll rephrase into shorter, clearer sentences.